src/lib/utilities/search-command-helpers.ts

```diff
@@ -0,0 +1,36 @@
+import type { AniListResponse } from '#lib/apis/anilist/anilist-constants';
+import type { Kitsu } from '#lib/apis/kitsu/Kitsu';
+import { LanguageKeys } from '#lib/i18n/LanguageKeys';
+import { SlashCommandSubcommandBuilder } from '@discordjs/builders';
+import { applyLocalizedBuilder } from '@skyra/http-framework-i18n';
+
+export function buildMangaSubcommand(name: 'kitsu' | 'anilist') {
+	return buildSubcommand(name, 'manga');
+}
+
+export function buildAnimeSubcommand(name: 'kitsu' | 'anilist') {
+	return buildSubcommand(name, 'anime');
+}
+
+export function checkIsKitsuSubcommand(subCommand: string | null, _value?: Kitsu.KitsuResult | AniListResponse): _value is Kitsu.KitsuResult {
```

Comment:
Overloads help documenting unused parameters nicely.
```suggestion
export function checkIsKitsuSubcommand(subCommand: string | null, value?: Kitsu.KitsuResult | AniListResponse): value is Kitsu.KitsuResult;
export function checkIsKitsuSubcommand(subCommand: string | null): boolean {
```